source/jormungandr/jormungandr/realtime_schedule/cleverage.py

```diff
@@ -29,7 +29,7 @@
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
 from __future__ import absolute_import, print_function, division
-from jormungandr.realtime_schedule.realtime_proxy import RealtimeProxy
+from jormungandr.realtime_schedule.realtime_proxy import RealtimeProxy, RealtimeProxyError
```

Comment:
How was it working previously if it wasn't imported? Why is it necessary now?

Reply:
Forgotten only in this file. All the oher files have this import : https://github.com/CanalTP/navitia/blob/b51ee3c1a5d166623f2a08f7e9243d2cb0cc9004/source/jormungandr/jormungandr/realtime_schedule/synthese.py#L34